docs/UserGuide.md

```diff
@@ -85,13 +85,13 @@ If you want a detailed look into each of the features that CampusCompanion has,
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+   * `list` : Lists all assignments and internships.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Campus Companion.
+   * `add-a n/CS2100 Assignment 1 e/2023-10-29` : Adds an assignment named `CS2100 Assignment 1` to the Campus Companion.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+   * `delete-a 3` : Deletes the 3rd assignment shown in the current list.
```

Comment:
Can change to `the assignment with index 3`. cus it might not necessarily be the third in the list